src/routes/Landing.tsx

```diff
@@ -0,0 +1,338 @@
+      </Flex>
+      <Flex as="main" flexDirection="column">
+        <Section
+          textAlign={["left", null, "center"]}
+          maxWidth="100ch"
+          margin="auto"
+          py={[24, null, null, 120, 150]}
+          px={6}
+        >
+          <Text as="h1" fontSize={[7, 8, 9, 10]} lineHeight={7}>
+            Retro tool,{" "}
+            <Box position="relative" as="span">
+              simple and effective{" "}
+              <Underline display={["none", null, null, "block"]} />
+            </Box>{" "}
+            retrospectives for your team
+          </Text>
+          <Box mt={6} mb={[8, null, 10]} mx="auto" maxWidth="60ch">
+            <Text fontSize={[5, null, 6]} color="grey">
+              Collaborate with your remote team to improve what you do, guided
+              by a simple workflow. And for free!
+            </Text>
+          </Box>
+          <Flex mb={2} justifyContent="center" textAlign="center">
+            {/*
+            // @ts-ignore */}
+            <Button
+              as="a"
+              href="/new"
+              variant="landingPrimary"
+              fontSize={[3, null, 4]}
+              tabIndex={1}
+              px={[4, null, 7]}
+            >
+              Create a new retro
+            </Button>
+            {/*
+            // @ts-ignore */}
+            <Button
+              as="a"
+              href="#how-it-works"
+              variant="landingSecondary"
+              ml={5}
+              tabIndex={2}
+              fontSize={[3, null, 4]}
+              px={[4, null, 7]}
+            >
+              See how it works
+            </Button>
+          </Flex>
+          <Text
+            fontSize={[3, null, 4]}
+            color="grey"
+            mt={7}
+            mb={[4, null, null, 0]}
+          >
+            Already{" "}
+            <Text fontSize="inherit" color="dark" as="span" fontWeight="bold">
+              1.259 retros
+            </Text>{" "}
+            and counting.
+          </Text>
+        </Section>
+        <Section bg="violet" color="white" py={[8, null, null, 80, 100]} px={7}>
+          <Box maxWidth="100ch" margin="auto">
+            <Text as="h2" hidden>
+              Main features
+            </Text>
+            <Flex justifyContent="space-between" flexWrap="wrap">
+              <Box
+                maxWidth={["100%", null, "45%", "20%"]}
+                mb={[9, 0]}
+                height={["auto", 250, "auto"]}
+              >
+                <Mobile height={32} />
+                <Text as="h3" mt={5} mb={4} textStyle="landingHighlightsTitle">
+                  Easy peasy
+                </Text>
+                <Text textStyle="landingHighlightsText">
+                  A mobile friendly UI with a clean and fast workflow that
+                  doesn't even need a tutorial.
+                </Text>
+                <Text textStyle="landingHighlightsText" mt={4}>
+                  Did we say already that it's free?
```

Comment:
what do you think of:

Free as in _freedom_, and as in _gratis_.

Reply:
Do you think is more understandable?! not for me at least!

Reply:
The tone in the original is not my favourite to be honest, but no strong opinions here 👍 However I'd correct it to `Did we already say that it's free?`